tests/e2e/support/objects/app-files/link/actions.ts

```diff
@@ -14,6 +14,45 @@ export type createLinkArgs = {
   via: 'SIDEBAR_PANEL' | 'QUICK_ACTION'
 }
 
+export type editLinkArgs = {
+  page: Page
+  resource: string
+  oldName: string
+  newName: string
+  role: string
+  dateOfExpiration: string
+  password: string
+}
+
+const fillPublicLinK = async (page, name, role, dateOfExpiration, password): Promise<void> => {
```

Comment:
the K at the end should be lower-case.
Please also use an object arg in the function which makes a future refactoring more easy.